app/views/users/show.html.erb

```diff
@@ -1 +1,20 @@
-User page
\ No newline at end of file
+<div>
+  <div>
+    <h1><%= @user.name %></h1>
+    <h4>Number of posts: <%= @user.posts.count %></h4>
+  </div>
+  <div>
+    <h4><strong>Bio</strong></h4>
+    <p><%= @user.bio %></p>
+  </div>
+  <div>
+    <ul>
+      <% @posts.each do |post| %>
+        <li>
+          <h4>Post: #<%= post.id %></h4>
+          <p><%= post.text %></p>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+</div>
```

Comment:
- [ ] User show page is required to a button that has `See All Posts` text that opens  user's post index page. That is yet to be implemented. Kindly fix that. It should look something similar to this. :point_down: 

```
<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
``` 

### Expected Page
![all-posts](https://github.com/Gh-Reza/BlogApp/assets/26597432/def87b13-9f0a-4ae1-973e-14ea0b6b82a5)